proposal: Team Monitoring -> Team Experience Analytics (#6955)
PostHog/posthog.com: contents/handbook/small-teams/monitoring/index.mdx

---

> title: Team Investigation

Reviewer: ```suggestion
title: Team Experience Analytics
```

> title: Experience Analytics Team

Reviewer: ```suggestion
title: Mojo Dojo Casa Team
```

Reviewer: ```suggestion
title: Qualitative Analytics Team
```

> title: Experience Analytics Team

Reviewer 2: `Investigation` and `Sherlock` would be my favorite as well.
I had to google `Experience Analytics` and I was not the only one, besides that, the definition found on Google does not meet our mission, so meh.

sidebar: Handbook
showTitle: true
hideAnchor: false
---

<TeamMembers />

## Mission

> We help customers understand _"the why"_ behind data and metrics tracked elsewhere in PostHog. Our tools help customers re-create and analyze the experience of individual users.
> 
> We build the heads up display for your product

Reviewer 2: ```suggestion
We help customers understand _"the why"_ behind data and metrics in PostHog. Our tools help customers re-create, analyze, and understand the experience of individual users.

We build the [heads-up](https://en.wikipedia.org/wiki/Head-up_display) display for your product
```

Reviewer: Ah, I missed the change in the larger thread, sorry

> We build the [heads-up](https://en.wikipedia.org/wiki/Head-up_display) display for your product

Reviewer: ```suggestion
We build the [heads-up display](https://en.wikipedia.org/wiki/Head-up_display) for your product
```

> - **Visibility** leads to **investigation** which leads to **understanding**.

Reviewer 2: I'd go with:

> We help customers understand _"the why"_ behind data and metrics in PostHog. Our tools help customers re-create, analyze, and understand the experience of individual users.
> 	
> We build the [heads-up](https://en.wikipedia.org/wiki/Head-up_display) display for your product

## Q4 2023 Goals

### 🎯 Goal 1 : Project "NoteForce Exploration 3000" (Ben, David)

> Why? Pairing up with Michael and Thomas from Team Product Analytics to focus on not just releasing Notebooks, but integrating them seamlessly with PostHog 3000 to make it a great experience to explore any and all data across PostHog.

Reviewer 2: ```suggestion
Why? Pairing up with Michael, Thomas and Cory from Team Product Analytics to focus on not just releasing Notebooks, but integrating them seamlessly with PostHog 3000 to make it a great experience to explore any and all data across PostHog.
```
